src/components/site/Breadcrumbs.md

```diff
@@ -0,0 +1,84 @@
+### Breadcrumbs Light
+
+```jsx
+import Breadcrumbs from "./Breadcrumbs";
```

Comment:
not blocking but you should need this import

Reply:
should not*

Reply:
Updated and committed. The import for /Breadcrumbs has been removed from the .md file.